app/models/user.js

```diff
@@ -0,0 +1,32 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      firstName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      lastName: {
+        type: DataTypes.STRING,
+        allowNull: false
+      },
+      email: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true,
+        validate: {
+          isEmail: true
+        }
+      },
+      password: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    { underscored: true }
```

Comment:
Two comments here: 
- You should add the tableName: `tableName: 'users'`
- Take a look at the `paranoid` option. It's very useful